### ISSUE TITLE
missing package mip

### ISSUE THREAD

**Issue by HMRLKE** [flipz357/smatchpp#14]
It seems that there is an unresolved dependency of the repo:

```
ModuleNotFoundError                       Traceback (most recent call last)
[/usr/local/lib/python3.10/dist-packages/smatchpp/solvers.py](https://localhost:8080/#) in __init__(self, max_seconds, backup_solver)
    370         try:
--> 371             import mip
    372             self.mip = mip

ModuleNotFoundError: No module named 'mip'

During handling of the above exception, another exception occurred:

ModuleNotFoundError                       Traceback (most recent call last)
1 frames
[<ipython-input-38-72595a466214>](https://localhost:8080/#) in <cell line: 20>()
     18 # Initialize Smatchpp components
     19 graph_standardizer = amrtools.AMRStandardizer()
---> 20 ilp = solvers.ILP()
     21 measure = Smatchpp(alignmentsolver=ilp, graph_standardizer=graph_standardizer)
     22 

[/usr/local/lib/python3.10/dist-packages/smatchpp/solvers.py](https://localhost:8080/#) in __init__(self, max_seconds, backup_solver)
    372             self.mip = mip
    373         except ModuleNotFoundError:
--> 374             raise ModuleNotFoundError("Module mip not found, please install mip \
    375                                        we used version 1.13.0")
    376         return None

ModuleNotFoundError: Module mip not found, please install mip
```

**Comment by flipz357** [comment]
Hi @HMRLKE thanks for filing this issue!

mip needs to be installed via pip. See the instruction here (it's just one line): https://github.com/flipz357/smatchpp#requirements

Now that you mention this, I think I should put it into the setup, so that it is always installed by default.

**Comment by flipz357** [comment]
Issue is solved in 1.7.0! Now mip is installed by default. 

Thanks for filing!
